### PR TITLE
opentelemetry docs update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/install-collector-macos.md [hunk 1]
```diff
@@ -28,10 +28,21 @@ Versions Supported
...
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **macOs** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in your system terminal where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/macos-terminal.png')} alt="execute command in terminal" />
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/install-collector-macos.md:42]:
```suggestion
1. Copy the command and execute it in your system terminal where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/macos-terminal.png')} alt="execute command in terminal" />
```

File: docs/send-data/opentelemetry-collector/install-collector-linux.md [hunk 2]
```diff
@@ -28,10 +28,21 @@ Supported Versions
...
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **Linux** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal.png')} alt="linux-terminal" style={{border: '1px solid black'}} width="900"/>
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in your system terminal where the collector needs to be installed.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal-installation.png')} alt="execute command in terminal" width="900"/>
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/install-collector-linux.md:42]:
```suggestion
1. Copy the command and execute it in your system terminal where the collector needs to be installed.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal-installation.png')} alt="execute command in terminal" width="900"/>
```

File: docs/send-data/opentelemetry-collector/install-collector-windows.md [hunk 3]
```diff
@@ -24,10 +24,21 @@ Supported Versions
...
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **Windows** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/windows.png')} alt="widows-terminal" style={{border: '1px solid black'}} width="900"/>
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in the powershell where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/windows-ui-install-6.png')} alt="windows-ui-installation-6.png" width="900" />
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/install-collector-windows.md:38]:
```suggestion
1. Copy the command and execute it in the powershell where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/windows-ui-install-6.png')} alt="windows-ui-installation-6.png" width="900" />
```

**Review by jpipkin1 — APPROVED** [review]
Looks very good, @JV0812. Approved.

File: docs/send-data/opentelemetry-collector/install-collector-linux.md [hunk 4]
```diff
@@ -28,10 +28,21 @@ Supported Versions
...
+* [UI Installation](#ui-installation)
 * [Install script](#install-script)
-* [UI Installation via App Catalog](#ui-installation-via-app-catalog)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **Linux** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal.png')} alt="linux-terminal" style={{border: '1px solid black'}} width="900"/>
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-linux.md:39]:
```suggestion
1. On the left panel, select **Linux** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal.png')} alt="linux-terminal" style={{border: '1px solid black'}} width="900"/>
```

File: docs/send-data/opentelemetry-collector/install-collector-linux.md [hunk 5]
```diff
@@ -28,10 +28,21 @@ Supported Versions
...
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in your system terminal where the collector needs to be installed.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/linux-terminal-installation.png')} alt="execute command in terminal" width="900"/>
+1. Wait for the installation to complete.
+1. Click **Next** to complete the installation process.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-linux.md:44]: is this correct?

```suggestion
1. Wait for the installation process to complete, then click **Next** to proceed.
```

File: docs/send-data/opentelemetry-collector/install-collector-macos.md [hunk 6]
```diff
@@ -28,10 +28,21 @@ Versions Supported
...
+* [UI Installation](#ui-installation)
 * [Install script](#install-script)
-* [UI Installation via App Catalog](#ui-installation-via-app-catalog)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **macOs** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-macos.md:39]:
```suggestion
1. On the left panel select **macOS** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```

File: docs/send-data/opentelemetry-collector/install-collector-macos.md [hunk 7]
```diff
@@ -28,10 +28,21 @@ Versions Supported
...
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in your system terminal where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/macos-terminal.png')} alt="execute command in terminal" />
+1. Wait for the installation to complete.
+1. Click **Next** to complete the installation process.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-macos.md:44]: is this correct? @angadrandhawa1 @ajaiswals 

```suggestion
1. Wait for the installation process to complete, then click **Next** to proceed.
```

File: docs/send-data/opentelemetry-collector/install-collector-windows.md [hunk 8]
```diff
@@ -24,10 +24,21 @@ Supported Versions
...
+* [UI Installation](#ui-installation)
 * [Install script](#install-script)
-* [UI Installation via App Catalog](#ui-installation-via-app-catalog)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel select **Windows** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/windows.png')} alt="widows-terminal" style={{border: '1px solid black'}} width="900"/>
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-windows.md:35]:
```suggestion
1. On the left panel, select **Windows** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/windows.png')} alt="widows-terminal" style={{border: '1px solid black'}} width="900"/>
```

File: docs/send-data/opentelemetry-collector/install-collector-windows.md [hunk 9]
```diff
@@ -24,10 +24,21 @@ Supported Versions
...
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy command and execute it in the powershell where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/windows-ui-install-6.png')} alt="windows-ui-installation-6.png" width="900" />
+1. Wait for the installation to complete.
+1. Click **Next** to complete the installation process.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-windows.md:40]: is this correct? 

```suggestion
1. Wait for the installation process to complete, then click **Next** to proceed.
```

File: sidebars.ts [hunk 10]
```diff
@@ -84,7 +84,7 @@ module.exports = {
           collapsed: true,
           link: {type: 'doc', id: 'send-data/opentelemetry-collector/index'},
           items: [
-            'send-data/opentelemetry-collector/quickstart',
+            //'send-data/opentelemetry-collector/quickstart',
```
Comment (kimsauce) [inline sidebars.ts:87]:
```suggestion
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting, style

File: docs/send-data/opentelemetry-collector/install-collector-macos.md (hunk 6 above)
Comment (JV0812) [reply docs/send-data/opentelemetry-collector/install-collector-macos.md:39]: It is an UI term, so maintaining macOs as it is.

**Comment by JV0812** [comment]
Thanks Kim, I have fixed your comments.

**Comment by JV0812** [comment]
Thanks John, I have fixed your comments.

File: docs/send-data/opentelemetry-collector/install-collector-macos.md [hunk 11]
```diff
@@ -28,10 +28,20 @@ Versions Supported
...
+* [UI Installation](#ui-installation)
 * [Install script](#install-script)
-* [UI Installation via App Catalog](#ui-installation-via-app-catalog)
 * [Manual step-by-step installation](#manual-step-by-step-installation)
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel, select **macOs** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-macos.md:39]:
```suggestion
1. On the left panel, select **macOS** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/macOs.png')} alt="macOs-terminal" style={{border: '1px solid black'}} width="900"/>
```

File: docs/send-data/opentelemetry-collector/install-collector-windows.md [hunk 12]
```diff
@@ -24,10 +24,21 @@ Supported Versions
...
 
+### UI Installation
+
+1. In Sumo Logic, select **Manage Data** > **Collection** > **OpenTelemetry Collection**.
+1. On the OpenTelemetry Collection page, click **Add Collector**.
+1. On the left panel, select **Windows** as the platform.<br/> <img src={useBaseUrl('img/send-data/opentelemetry-collector/windows.png')} alt="widows-terminal" style={{border: '1px solid black'}} width="900"/>
+1. Select/create installation token and customize your tags.
+1. (Optional) Select the **Auto Configure Host and Process metrics data collection** checkbox to collect host and process metrics.
+1. Copy the command.
+1. Open powershell and execute the command in the powershell where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/windows-ui-install-6.png')} alt="windows-ui-installation-6.png" width="900" />
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector-windows.md:39]:
```suggestion
1. Open PowerShell and execute the command in the PowerShell where the collector needs to be installed.<br/><img src={useBaseUrl('img/send-data/opentelemetry-collector/windows-ui-install-6.png')} alt="windows-ui-installation-6.png" width="900" />
```

File: sidebars.ts [hunk 13]
```diff
@@ -84,7 +84,6 @@ module.exports = {
           collapsed: true,
           link: {type: 'doc', id: 'send-data/opentelemetry-collector/index'},
           items: [
-            'send-data/opentelemetry-collector/quickstart',
```
Comment (kimsauce) [inline sidebars.ts:87]: Why is the quickstart being deleted @JV0812 ?

File: docs/send-data/opentelemetry-collector/install-collector-macos.md (hunk 11 above)
Comment (kimsauce) [reply docs/send-data/opentelemetry-collector/install-collector-macos.md:39]: @JV0812 I understand that "macOs" matches the UI, which is incorrect. I've reported the error [here](https://sumologic.slack.com/archives/C1PB1DMB3/p1687548320959719), and the devs are fixing, so it's ok to change to "macOS".

File: sidebars.ts (hunk 13 above)
Comment (JV0812) [reply sidebars.ts:87]: @kimsauce The content in the quickstart does not go good with the current UI and the work is going on to update the UI accordingly. Also, I stashed this line to hide the doc from the side, but as per your comment I just deleted the path.

File: docs/send-data/opentelemetry-collector/install-collector-macos.md (hunk 11 above)
Comment (JV0812) [reply docs/send-data/opentelemetry-collector/install-collector-macos.md]: @kimsauce Thanks for reporting these UI changes.

**Review by kimsauce — APPROVED** [review]

File: sidebars.ts (hunk 13 above)
Comment (kimsauce) [reply sidebars.ts:87]: Oh I see what you're saying. In this case it's ok to hide it like you did before, and maybe add a note (commented out) at the top of the quickstart like this:

```
---
id: quickstart
title: OpenTelemetry Collector Quickstart
sidebar_label: Quickstart
description: Get up and running quickly with the Sumo Logic OpenTelemetry Collector.
---

<!-- temporarily pulled from docs; UI screenshots being updated -->

import useBaseUrl from '@docusaurus/useBaseUrl';

Sumo Logic makes it easy to get started with OpenTelemetry by providing a custom distribution of the OpenTelemetry collector, which can be used to send metrics, logs, and traces directly to our platform.
```

**Pull request merged**